nlp/2021_06_29_long_range_transformers/README.md

```diff
@@ -0,0 +1,13 @@
+# Long range transformers
+
+In recent research there has been an extensive study for improving the calculation of attention in transformer architectures. Mostly for improving their capacity to handle longer token sequences. 👊
```

Comment:
Bit of a weird opening sentence, I sounds like there was this one multi-year study. I would rephrase it more to denote that it's a clear trend of the past year(s) or smting, in the opening statement